alpha/template/semver/types.go

```diff
@@ -62,10 +63,12 @@ func (b byChannelPriority) Swap(i, j int) { b[i], b[j] = b[j], b[i] }
 
 type streamType string
 
+const invalidStreamType streamType = "INVALID"
```

Comment:
I might be misinterpreting, but is the intention here to have a sentinel value for "lower priority than other stream types"?

If so, naming this "invalid" strikes me as potentially confusing. With this name, I was expecting to see some parsing code that referenced this value in an error case.

What do you think about dropping this constant and using empty string `""` (which is the default value for `streamType`) to represent the lowest priority stream type?

Reply:
Thanks for the quick review @joelanford!

Oops.  Debugging implementation detail leakage.  Yes, it was a sentinel, and I was using it to ensure that there were no cases where that value wasn't initialized (and because some of the debugging involved fmt.Printf, it was better to see 'INVALID' than "" :) ). 
It's not necessary anymore.  Will remove.